src/main/java/io/cdap/e2e/pages/actions/CdfStudioActions.java

```diff
@@ -498,11 +502,38 @@ public static void closeStatusBanner() {
 
   /**
    * Click on the element and move.
+   *
    * @param pluginName plugin title.
-   * @param xOffset horizontal move offset.
-   * @param yOffset vertical move offset.
+   * @param xOffset    horizontal move offset.
+   * @param yOffset    vertical move offset.
    */
   public static void movePlugin(String pluginName, int xOffset, int yOffset) {
     ElementHelper.dragAndDropByOffset(CdfStudioLocators.locatePluginNodeInCanvas(pluginName), xOffset, yOffset);
   }
+
+  /**
+   * Click on the Upload Button
+   */
+  public static void clickOnAddEntityButton() {
+    ElementHelper.clickOnElement(CdfStudioLocators.addEntityButton);
+  }
+
+  /**
+   * Clicks on the "Fix All" button if displayed (available in CDAP 6.10.0+).
+   */
+
+  public static void clickOnFixAllButton() {
```

Comment:
Rename: clickOnFixAllButtonIfDisplayed

Reply:
Renamed to 'clickOnFixAllButtonIfDisplayed'